src/evalvars.c

```diff
@@ -161,6 +161,7 @@ static struct vimvar
     {VV_NAME("t_typealias",	 VAR_NUMBER), NULL, VV_RO},
     {VV_NAME("t_enum",		 VAR_NUMBER), NULL, VV_RO},
     {VV_NAME("t_enumvalue",	 VAR_NUMBER), NULL, VV_RO},
+    {VV_NAME("cmdcomplete",	 VAR_NUMBER), NULL, VV_RO},
```

Comment:
should this be bool? And then changing the documentation to `v:true`/`v:false` instead?

Reply:
I'm not sure. There has never been a bool v: variable before.

Reply:
Actually I think it makes sense. I'll change it.